### PR TITLE
Docs stupid simple stuff

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

File: docs/src/gui/gladevcp.adoc [hunk 1]
```diff
@@ -2659,7 +2659,7 @@ This should set debug to 2 and confirm that your module actually did it.
 
 === Persistent variables in GladeVCP
 
-A annoying aspect of GladeVCP in its earlier form and pyvcp is the
+A annoying aspect of GladeVCP in its earlier form and PyVCP is the
```
Comment (hansu) [inline docs/src/gui/gladevcp.adoc:2662]:
```suggestion
An annoying aspect of GladeVCP in its earlier form and PyVCP is the
```

File: docs/src/gui/gladevcp.adoc [hunk 2]
```diff
@@ -357,7 +357,7 @@ using a run-in-place setup.
 [[gladevcp:embedding-tab]]
 === Embedding as a Tab
 
-To do so, edit your .ini file and add to the DISPLAY and HAL sections of INI
+To do so, edit your .ini (INI) file and add to the DISPLAY and HAL sections of INI
```
Comment (hansu) [inline docs/src/gui/gladevcp.adoc:360]:
```suggestion
To do so, edit your INI file and add to the DISPLAY and HAL sections of INI
```
to have it consistent

File: docs/src/plasma/qtplasmac.adoc [hunk 3]
```diff
@@ -1745,11 +1745,11 @@ The minimum percentage allowed is 10%, values below this will be set to 10%.
 
 The maximum percentage allowed is 100%, values above this will be set to 100%.
 
-If the user intends to use this feature it would be prudent to add *M68 E3 Q0* to both the preamble and postamble of the G-Code program so the machine starts and ends in a known state.
+If the user intends to use this feature it would be prudent to add *M68 E3 Q0* to both the preamble and postamble of the G-code program so the machine starts and ends in a known state.
 
-IMPORTANT: *G-CODE THC* AND *VELOCITY BASED THC* ARE NOT ABLE TO BE USED IF *CUTTER COMPENSATION* IS IN EFFECT; AN ERROR MESSAGE WILL BE DISPLAYED.
+IMPORTANT: *G-cODE THC* AND *VELOCITY BASED THC* ARE NOT ABLE TO BE USED IF *CUTTER COMPENSATION* IS IN EFFECT; AN ERROR MESSAGE WILL BE DISPLAYED.
```
Comment (petterreinholdtsen) [inline docs/src/plasma/qtplasmac.adoc:1750]: "G-cODE" do not look right.

File: docs/src/plasma/qtplasmac.adoc [hunk 4]
```diff
@@ -1797,15 +1797,15 @@ It is important to note that <<qt_velocity-reduction, Velocity Reduction>> affec
...
+LinuxCNC (QtPlasmaC) has the ability to automatically adjust the cut path of the current program by the amount specified in Kerf Width of the selected material's Cut Parameters. This is helpful if the G-code is programmed to the nominal cut path and the user will be running the program on different thickness materials to help ensure consistently sized parts.
 
 To use cutter compensation the user will need to use G41.1, G42.1 and G40 with the kerf width HAL pin:
 
 * *G41.1 D#<_hal[plasmac_run.kerf-width-f]>* : offsets torch to the left of the programmed path
 * *G42.1 D#<_hal[plasmac_run.kerf-width-f]>* : offsets torch to the right of the programmed path
 * *G40* turns the cutter compensation off
 
-IMPORTANT: IF *CUTTER COMPENSATION* IS IN EFFECT *G-CODE THC*, *VELOCITY BASED THC* AND *OVER CUT* ARE NOT ABLE TO BE USED; AN ERROR MESSAGE WILL BE DISPLAYED.
+IMPORTANT: IF *CUTTER COMPENSATION* IS IN EFFECT *G-cODE THC*, *VELOCITY BASED THC* AND *OVER CUT* ARE NOT ABLE TO BE USED; AN ERROR MESSAGE WILL BE DISPLAYED.
```
Comment (petterreinholdtsen) [inline docs/src/plasma/qtplasmac.adoc:1808]: "G-cODE" -> "G-CODE"?

**Comment by petterreinholdtsen** [comment]
This change look good, and I believe it should go in as soon as the two typos are fixed.

File: docs/src/gui/gladevcp.adoc (hunk 2 above)
Comment (hansu) [reply docs/src/gui/gladevcp.adoc:360]: ?

Comment (smoe) [reply docs/src/gui/gladevcp.adoc:360]: At some point you need to introduce that .ini and INI files are the same thing.

Comment (hansu) [reply docs/src/gui/gladevcp.adoc:360]: Then it should be introduced in every document where "INI file" and similar is used. But that should be clear to people who can operate a pc, that an INi file is a file with the extension "ini".
Even in https://linuxcnc.org/docs/devel/html/config/ini-config.html it is not mentioned, that it is a file with .ini extension...

**Comment by smoe** [comment]
I would prefer adding it to the ini-config file over removing the extra clarity here. The same for HAL files.

**Comment by smoe** [comment]
I gave in - added respective patch and rebased to master.

**Comment by hansu** [comment]
> I gave in - added respective patch and rebased to master.

As this patch is about details, we also have to speak in details 😉 

If we want to explain/introduce that (what is obvious IMHO) I would suggest it this way (in general):
`To do so, edit your INI file (.ini) and add to the DISPLAY and HAL sections of INI`

**Comment by smoe** [comment]
Except that this patch removes that detail so we do not need to talk about it - in this thread. I want to move on, please.

**Comment by hansu** [comment]
Sure, sorry 😇

**Pull request merged**